workers.tf

```diff
@@ -384,3 +397,63 @@ resource "aws_iam_role_policy_attachment" "workers_additional_policies" {
   role       = aws_iam_role.workers[0].name
   policy_arn = var.workers_additional_policies[count.index]
 }
+
+resource "aws_iam_role_policy_attachment" "workers_autoscaling" {
```

Comment:
@ruccoon Did you add the following changes? If so, can I know why? We need to use this to test on staging though.

Reply:
they are from your branch
https://github.com/viperKiran/terraform-aws-eks/blob/8d725d06aae549bc63afe33bb07b82ff9cdb71c3/workers.tf#L371-L425